lib/index.js

```diff
@@ -359,6 +359,20 @@ Object.defineProperty(Nightwatch, 'browser', {
   }
 });
 
+Object.defineProperty(Nightwatch, 'app', {
```

Comment:
Nit: we can use `defineProperties`, and define both `app` and `browser`

Reply:
IIUC [`Object.defineProperties`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperties#using_object.defineproperties) cannot be used to add the same descriptor to two properties, but is used to add all the properties in one go. But since all the properties are added to `Nightwatch` separately in this file, it makes more sense to me to use `defineProperty` only just to have a consistent pattern through the file.

But if the problem is with the repetition of the same descriptor for the two properties, we can maybe do something like this?
```js
const nightwatchApiDescriptor = {
  configurable: true,
  get() {
    if (global.browser) {
      return global.browser;
    }

    const err = new TypeError('Nightwatch client is not yet available.');
    err.addDetailedErr = true;

    throw err;
  }
};

Object.defineProperty(Nightwatch, 'browser', nightwatchApiDescriptor);

Object.defineProperty(Nightwatch, 'app', nightwatchApiDescriptor);
```

Reply:
Yes, can you do the refactoring to avoid repeating the code?

Reply:
Done